packages/webviz-core/src/panels/ThreeDimensionalViz/commands/utils/index.js

```diff
@@ -36,76 +36,113 @@ export function setRgba(buffer: Uint8Array, index: number, color: tinycolor) {
   buffer[index + 3] = rgba255.a;
 }
 
+// Ported from Rviz' Map palette
```

Comment:
I'm fine with this but it seems like it could be acheived with far fewer changes I think

As far as I can tell cruise is correct on the grey and green map pallettes (albeit they use lime instead of green)

The `illegal negative (char) values in shades of red/yellow` are a different calculation 
```
tinycolor.fromRatio({ r: 1.0, g: t, b: 0, a: 1 }));
```
as far as I can tell.

---
Then we knock off their illegal negatives and we're done right?

Reply:
This is a straight port of the Rviz code.  The grey colors were slightly off

Reply:
Wasn't theirs though? has the same comments and stuff 

Wait how.. isn't 
```
const t = 1 - i / 100;
```

the same as

```
const v = 255 - (255 * i) / 100;
```

Like their just scaled differently right?

Reply:
I guess.  I just want the exact same code that Rviz uses, not whatever Cruise cooked up.  If you really care about this, I can change it back, but I wanted the same code so that anyone can see at a glance that it's identical to Rviz.